pkg/mountutil/volumestore/volumestore.go

```diff
@@ -83,13 +83,29 @@ func (vs *volumeStore) Create(name string, labels []string) (*native.Volume, err
 	}
 	volPath := filepath.Join(vs.dir, name)
 	volDataPath := filepath.Join(volPath, DataDirName)
-	fn := func() error {
+	volFilePath := filepath.Join(volPath, volumeJSONFileName)
+	fn := func() (err error) {
 		if err := os.Mkdir(volPath, 0700); err != nil {
 			return err
 		}
+		defer func() {
+			if err != nil {
+				// ignore cleanup error
+				os.Remove(volPath)
+			}
+		}()
 		if err := os.Mkdir(volDataPath, 0755); err != nil {
 			return err
 		}
+		defer func() {
+			if err != nil {
+				// ignore cleanup error
```

Comment:
error shouldn't be ignored. can we print log for this?

Reply:
> error shouldn't be ignored. can we print log for this?

I don't print log for cleanup error as some other cleanup functions `defer os.Remove` in the code.
Of course, I can also add some logs for these remove errors